thoth/user_api/api_v1.py

```diff
@@ -1139,3 +1139,23 @@ def _do_get_image_metadata(
         error_str = str(exc)
 
     return {"error": error_str, "parameters": locals()}, status_code
+
+
+def get_imported_packages(name: str):
```

Comment:
`import_package_name`?

Reply:
> `import_package_name`?

fixed in 0ada6fd042629810b21dcec94b4be45bd7b93e6b